__tests__/flavored-parsers.test.js

```diff
@@ -99,4 +99,34 @@ describe('Parse RDMD Syntax', () => {
     const ast = process(mdx);
     expect(ast.children).toHaveLength(3);
   });
+
+  it('More foolish disregard for newlines.', () => {
+    const mdx = '```\nfoo\n```\nOops\n```\nbar\n```';
+    const ast = process(mdx);
+
+    expect(ast.children.map(c => c.type)).toStrictEqual(['code', 'paragraph', 'code']);
```

Comment:
Whoah, I hadn't thought to test block type ordering like this; love the legibility of it.